ci/install-package-dependencies.sh

```diff
@@ -86,6 +86,20 @@ echo "$EDATOOLS_REPO" > "$TMPDIR/obs.list"
 sudo mv "$TMPDIR/obs.asc"  /etc/apt/trusted.gpg.d/obs.asc
 sudo mv "$TMPDIR/obs.list" /etc/apt/sources.list.d/edatools.list
 
+# Install gcc-8 and set it as the default
```

Comment:
nit: below commands seem to be installing `gcc-9` while the comment says `gcc-8`.

Reply:
Done